conference-finder/src/ConferenceItem.jsx

```diff
@@ -2,12 +2,18 @@ import React from "react";
 
 const ConferenceItem = ({ conference }) => {
   const { name, topic, dateTime, city } = conference;
+  const date = new Date(dateTime);
   return (
     <li>
       <div className="card">
         <h3>{name}</h3>
         <p>{topic}</p>
-        <p>{dateTime}</p>
+        <p>
+          {date.getDate()}/{date.getMonth() + 1}/{date.getFullYear()}
```

Comment:
Just a thought, as I've approved already before this commit. But I would recommend you do this logic of building your date on like 6 instead.

I'd also say that using a template literal instead of opening and closing curly braces is nicer to read!

Have a read of these docs, it's a library used to displaying and manipulating dates and is really useful: https://momentjs.com/docs/#/displaying/